julia-repl.el

```diff
@@ -270,7 +270,8 @@ Valid backends are currently:
      (setq julia-repl--terminal-backend (make-julia-repl--buffer-ansi-term)))
     ('vterm
      (require 'vterm)
-     (setq julia-repl--terminal-backend (make-julia-repl--buffer-vterm)))
+     (setq julia-repl--terminal-backend (make-julia-repl--buffer-vterm))
+     (add-to-list 'vterm-eval-cmds '("julia-repl--show" julia-repl--show)))
```

Comment:
What happens if a user does not have EmacsVterm.jl installed at all? Does this degrade gracefully, or do we get an error?

Should we have a customizable variable instead that determines whether EmacsVterm.jl integration is enabled?

Reply:
The code here allows vterm.el to call `julia-repl--show` when a special escape sequence is printed to the terminal. If EmacsVterm.jl is not installed, Julia will not send that escape sequence to the terminal and `julia-repl--show` will never be called. In other words, everything will work as without these changes.

I don't think we need an customizable variable now. The whole integration can be disabled by removing EmacsVterm.jl initialization from `startup.jl`.

We might need some customization later, if EmacsVterm.jl starts sending more things than just documentation. But the question will be whether to customize on Julia side or Emacs side.